plugins/certs/help-functions

```diff
@@ -34,6 +34,7 @@ fn-help-content() {
     certs:remove <app>, Remove an SSL Endpoint from an app
     certs:report [<app>] [<flag>], Displays an ssl report for one or more apps
     certs:rollback <app>, [NOT IMPLEMENTED] Rollback an SSL Endpoint for an app
+    certs:show <app> <crt|key>, Show the server.crt or server.key on stdout
```

Comment:
This description should be the same as the following two:

- https://github.com/dokku/dokku/pull/4041/files#diff-3cfb1097cf8847f24c036fe8f8128a8eR12
- https://github.com/dokku/dokku/pull/4041/files#diff-5d6d6d7dafc57ca6c4c764ec0861c169R8

Reply:
Done